### PR TITLE
Saas 14.5 dropdown extends sri boi

### PR DESCRIPTION
**Pull request by brboi**
Related: https://github.com/odoo/enterprise/pull/21035

(initial PR from sri : #75302)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/static/src/xml/report_stock_forecasted.xml [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
-    <div id="warehouse_filter" class="btn-group o_dropdown o_stock_report_warehouse_filter"
+    <div id="warehouse_filter" class="btn-group dropdown o_stock_report_warehouse_filter"
         t-if="displayWarehouseFilter">
-        <button type="button" class="o_dropdown_toggler btn btn-secondary dropdown-toggle"
+        <button type="button" class="dropdown-toggle btn btn-secondary dropdown-toggle"
             data-toggle="dropdown">
             <span class="fa fa-home"/> Warehouse: <t t-esc="active_warehouse['name']"/>
         </button>
-        <div class="dropdown-menu o_dropdown_menu o_filter_menu" role="menu">
+        <div class="dropdown-menu dropdown-menu o_filter_menu" role="menu">
```
Comment (FrancoisGe) [inline addons/stock/static/src/xml/report_stock_forecasted.xml:23]: It s ok ?

File: addons/web/static/src/core/dropdown/dropdown.scss [hunk 2]
```diff
@@ -1,144 +1,66 @@
...
-  > span {
-    min-width: max-content;
+.o-dropdown {
+  &--menu {
+    // Makes sure the dropdown menus are already in a
+    // fixed position before the positioning computations begin.
+    position: fixed;
+  }
+  /*!rtl:begin:ignore*/
+  &.dropbottom > .o-dropdown--menu,
```
Comment (brboi) [inline addons/web/static/src/core/dropdown/dropdown.scss:36]: Wrong rule here

**Review by brboi — COMMENTED** [review]
Wrong commit title/description

**Comment by brboi** [comment]
@stefanorigano OK for you if I squash the second commit ?

**Comment by stefanorigano** [comment]
> @stefanorigano OK for you if I squash the second commit ?
Sure 👍

File: addons/board/static/tests/dashboard_tests.js [hunk 3]
```diff
@@ -793,9 +795,9 @@ QUnit.test("save actions to dashboard", async function (assert) {
     // add this action to dashboard
     await toggleFavoriteMenu(webClient);
 
-    await testUtils.dom.click($(".o_add_to_board button.o_dropdown_toggler"));
+    await testUtils.dom.triggerEvent($(".o_add_to_board button.dropdown-toggle"), "mouseenter");
```
Comment (FrancoisGe) [inline addons/board/static/tests/dashboard_tests.js:798]: utilser le utils mouseEnter

File: addons/board/static/tests/dashboard_tests.js [hunk 4]
```diff
@@ -849,8 +852,8 @@ QUnit.test("save two searches to dashboard", async function (assert) {
 
     // Add it to dashboard
     await toggleFavoriteMenu(webClient);
-    await testUtils.dom.click($(".o_add_to_board button.o_dropdown_toggler"));
-    await testUtils.dom.click($(".o_add_to_board .o_dropdown_menu button"));
+    await testUtils.dom.triggerEvent($(".o_add_to_board button.dropdown-toggle"), "mouseenter");
```
Comment (FrancoisGe) [inline addons/board/static/tests/dashboard_tests.js:855]: utils mouseEnter

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo rebase-ff

**Pull request closed without merging**